styles/widgets/base/scheduler.less

```diff
@@ -442,6 +445,111 @@
     display: none;
 }
 
+.dx-scheduler-header-panel-cell.dx-scheduler-header-panel-current-time-cell {
+    box-shadow: inset 0px -2px 0px 0px @SCHEDULER_APPOINTMENT_BASE_COLOR;
+    color: @SCHEDULER_APPOINTMENT_BASE_COLOR;
+}
+
+.dx-scheduler-date-time-shader-all-day {
+    margin-left: @SCHEDULER_LEFT_COLUMN_WIDTH;
+    width: 100%;
+    background-color: @SCHEDULER_TIME_INDICATOR_BACKGROUND_COLOR;
+    position: absolute;
+    pointer-events: none;
+}
+
+.dx-scheduler-date-time-indicator {
+        background-color:  @SCHEDULER_APPOINTMENT_BASE_COLOR;
+        position: absolute;
+        box-shadow: @SCHEDULER_TIME_INDICATOR_SHADOW;
+
+        &:before {
+            font-size: @SCHEDULER_TIME_INDICATOR_FONT_SIZE;
+            color: @SCHEDULER_APPOINTMENT_BASE_COLOR;
+            text-shadow: @SCHEDULER_WORKSPACE_BACKGROUND_COLOR 1px -1px 3px;
+            position: absolute;
+            z-index: 1000;
+            margin-top: -15px;
+            margin-left: -12px;
+        }
+}
+
+.dx-scheduler-work-space-week, 
+.dx-scheduler-work-space-work-week,
+.dx-scheduler-work-space-day {
+    .dx-scheduler-date-time-indicator {
+        margin-left: @SCHEDULER_LEFT_COLUMN_WIDTH;
+        height: 1px;
+        .dx-icon-spinright;
+    }
+
+    .dx-scheduler-date-time-shader{
+        margin-left: @SCHEDULER_LEFT_COLUMN_WIDTH;
+    }
+
+    .dx-scheduler-date-time-shader-top,
+    .dx-scheduler-date-time-shader-bottom {
+         &:before {
+             width: inherit;
+             height: inherit;
+             content: "";
+             left: inherit;
+             position: absolute;
+             margin-left: @SCHEDULER_LEFT_COLUMN_WIDTH;
+             background-color:  @SCHEDULER_TIME_INDICATOR_BACKGROUND_COLOR;
+             pointer-events: none;
+         }
+    }
+
+    &.dx-scheduler-work-space-both-scrollbar {
+        .dx-scheduler-date-time-shader,
+        .dx-scheduler-date-time-indicator,
+        .dx-scheduler-date-time-shader-all-day {
+            margin-left: 0;
```

Comment:
Does it work correctly in rtl mode?

Reply:
Indicator behaviour in rtl mode was fixed for day/week views. There are some problems in timelines https://isc.devexpress.com/Thread/WorkplaceDetails/T561264